day1/exp15.java

```diff
@@ -8,93 +8,109 @@ public static void main(String[] args) {
 		Primelist.print(100);
 		Scanner sc=new Scanner(System.in);
 		int x=sc.nextInt();
-		Arm.check(x);
-		perfect.check(x);
-		palin.check(x);
-		
-	}
+		exp15.display(x);
 }
-class Primelist{
-	static void print(int b) {
-		System.out.println("Prime numbers between 1 to 100");
-		for(int i=2;i<b;i++) {
-		if(check(i)==1)
-			System.out.println(i);
-		
-		
+	static void display(int x) {
+		int flag=0;
+		if(CheckList.Armcheck(x)==1) {
```

Comment:
method naming should follow **camelCase** Armcheck() -> armCheck()

Learn about naming conventions - https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

and make the appropriate changes in all of the programs